content/blog/2019/07/2019-07-03-remoting-kafka-kubernetes-phase-1.adoc

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "Remoting over Apache Kafka plugin with Kafka launcher in Kubernetes"
+tags:
+- remoting
+- kafka
+- kubernetes
+- helm
+- developer
+- gsoc
+- gsoc2019
+author: longnguyen
+---
+
+= Remoting over Apache Kafka plugin with Kafka launcher in Kubernetes
+
+== Introduction
+
+Current version of Remoting over Apache Kafka plugin requires users to manually configure the entire system which includes Zookeeper, Kafka and remoting agents. It also doesn't support dynamic agent provisioning so scalability is harder to achieve. This project aims to solve two problems:
+
+. Out-of-the-box solution to provision Apache Kafka cluster.
+
+. Dynamic agent provisioning in a Kubernetes cluster.
+
+== Current State
+
+* Kubernetes connector with credentials supported.
+
+* Apache Kafka provisioning in Kubernetes feature is fully implemented.
+
+* Helm chart is partially implemented.
+
+== Apache Kafka provisioning in Kubernetes
+
+This feature is part of 2.0 version so it is not yet released officially. You can try out the feature by building directly from master branch:
```

Comment:
Hint: you can release `alpha` or `beta` version so that it becomes accessible from experimental UCs